addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -115,6 +115,14 @@ def _l10n_es_tbai_post_invoice_edi(self, invoice):
             if invoice.move_type == 'out_refund' and not invoice.reversed_entry_id._l10n_es_tbai_is_in_chain():
                 error_msg = _("TicketBAI: Cannot post a reversal move while the source document (%s) has not been posted", invoice.reversed_entry_id.name)
 
+            # Tax configuration check: In case of foreign customer we need the tax scope to be set
+            com_partner = invoice.commercial_partner_id
+            if com_partner.country_id.code not in ('ES', False) or (com_partner.vat or '').startswith("ESN"):
```

Comment:
Hm, where do you check that the tax scope is set?

Reply:
Indeed I forgot a piece of the condition, thanks